dspace/config/dspace.cfg

```diff
@@ -1459,10 +1459,18 @@ sherpa.romeo.apikey =
 #    org.dspace.content.authority.SolrAuthority = SolrAuthorAuthority
 
 # URL of ORCID API
-# Defaults to using the Public API (pub.orcid.org)
-orcid.api.url = https://pub.orcid.org/v2.1
+# Defaults to using the Public API V3 (pub.orcid.org)
+orcid.api.url = https://pub.orcid.org/v3.0
 orcid.url = https://orcid.org/
 
+# ORCID Credentials
```

Comment:
Should we provide a link or more information on where to obtain these ORCID credentials for your repository?  I'm assuming that this Id/secret can be obtained via either the "Member API" or "Public API" signup available here: https://orcid.org/content/register-client-application-0  Is that correct?